ocs_ci/framework/__init__.py

```diff
@@ -8,7 +8,7 @@
 """
 # Use the new python 3.7 dataclass decorator, which provides an object similar
 # to a namedtuple, but allows type enforcement and defining methods.
-import collections
+import collections.abc
```

Comment:
Should we use `from` here?

```suggestion
from collections import abc
```

Reply:
I just did `from collections.abc import Mapping` since only one ABC is used